### PR TITLE
MFTF tests coverage for Used In links && fix bug with category grid urlFIlterApplie

### LINKED ISSUE #1693: MFTF tests coverage for Used In links
### Description (*)
MFTF tests coverage for Used In links
See magento/adobe-stock-integration#1503: Add links to grids to the image details panel

### Preconditions
Have an image in media gallery that was inserted to page, block, category and product content

### Manual testing scenarios (*)
1. Open the Media Gallery
2. View details of the image
3. Click links in Used In section

### Expected Result
Corresponding grid is opened with the entities filtered by the image (asset)

### LINKED ISSUE #1700: "Uncaught ReferenceError" appears in dev console if add a new tag and place the mouse cursor over it
### Preconditions

1. Install Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Enhanced Media Gallery Enabled
4. Uploaded image 

### Steps to reproduce (*)
1. Select the uploaded image by clicking on it
2. Click "three dots", select **Edit**
3. Place the cursor on _New Keyword_ field and type in some text, "_test_" for example
4. Click **Add New Tag**
(the tag appeared in Tags drop-down)
5. Click on **Tags** drop-down to expand it
6. Place the mouse cursor over the previously added _test_ tag
Peek 2020-08-03 12-09

### Actual result (*)
:x: _Uncaught ReferenceError: $index is not defined at UiClass.mousemove (eval at createBindingsStringEvaluator (knockout.js:2982)_ error appears in dev console 
uncought_reference error

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1693: MFTF tests coverage for Used In links
2. Fixes magento/adobe-stock-integration#1700: "Uncaught ReferenceError" appears in dev console if add a new tag and place the mouse cursor over it

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by engcom-Golf** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

File: app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Test/AdminMediaGalleryCatalogUiVerifyUsedInLinkCategoryGridTest.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <description value="User can open each entity the asset is associated with in a separate tab to manage association"/>
+            <severity value="CRITICAL"/>
+            <group value="media_gallery_ui"/>
+        </annotations>
+        <before>
+            <createData entity="SimpleSubCategory" stepKey="category"/>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+        </before>
+        <after>
+        <actionGroup ref="AdminOpenStandaloneMediaGalleryActionGroup" stepKey="openMediaGallery"/>
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Test/AdminMediaGalleryCatalogUiVerifyUsedInLinkCategoryGridTest.xml]: Please correct the indents

File: app/code/Magento/MediaGalleryCatalogUi/view/adminhtml/layout/media_gallery_catalog_category_index.xml [hunk 2]
```diff
@@ -10,6 +10,11 @@
     <body>
         <referenceContainer htmlTag="div" htmlClass="media-gallery-category-container" name="content">
             <uiComponent name="media_gallery_category_listing"/>
+            <block class="Magento\Backend\Block\Template" template="Magento_Cms::url_filter_applier.phtml" name="category_list_url_filter_applier">
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryCatalogUi/view/adminhtml/layout/media_gallery_catalog_category_index.xml]: Please introduce a phtml file to the `MediaGalleryCatalogUi` module

File: app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js [hunk 3]
```diff
@@ -54,9 +55,33 @@ define([
         initialize: function () {
             this._super().initView();
 
+            this.removeMouseOverEvent();
+
             return this;
         },
 
+        /**
+         * Remove mousemove event from ui-select as it deprecated
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js]: I think we should remove the mousemove event from `ui-select` component. It's deprecated since 2016

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @Nazar65 ! Please see my comments

File: app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js (hunk 3 above)
Comment (Nazar65) [reply app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js]: ok

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

File: app/code/Magento/Ui/view/base/web/templates/grid/filters/elements/ui-select.html [hunk 4]
```diff
@@ -141,13 +141,12 @@
         </div>
         <div ifnot="options().length"
              class="admin__action-multiselect-empty-area">
-            <ul data-bind="html: emptyOptionsHtml"/>
+            <ul data-bind="text: emptyOptionsHtml"/>
```
Comment (sivaschenko) [inline app/code/Magento/Ui/view/base/web/templates/grid/filters/elements/ui-select.html]: Please revert this change, it is breaking backward compatibility

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the update @Nazar65 ! Please see my comment

**Comment by Nazar65** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-7972](https://jira.corp.magento.com/browse/ENGCOM-7972) has been created to process this Pull Request

**Comment by chalov-anton** [comment]
:heavy_check_mark: QA Passed

No error in dev console when placing mouse cursor over the added tag 

Test on Cucumber Studio - https://studio.cucumber.io/projects/131313/test-plan/folders/1320712/scenarios/3961351

**Comment by engcom-Kilo** [comment]
Failed Functional Tests B2B not related to the changes in this PR
